scripts/Synapse/updateECRDataStorePersonID.ipynb

```diff
@@ -0,0 +1,157 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": true
+        }
+      },
+      "outputs": [],
+      "source": [
+        "pip install psycopg2-binary azure-keyvault"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "from azure.identity import ManagedIdentityCredential\n",
+        "from azure.core.credentials import AccessToken\n",
+        "from azure.keyvault.secrets import SecretClient\n",
+        "import psycopg2\n",
+        "import time\n",
+        "\n",
+        "from delta.tables import *\n",
+        "from pyspark.sql.functions import *"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "ECR_DELTA_TABLE_FILE_PATH = \"/delta-tables/ecr_datastore\"\n",
+        "\n",
+        "# Set up authentication\n",
+        "class spoof_token:\n",
+        "    def get_token(*args, **kwargs):\n",
+        "        return AccessToken(\n",
+        "            token=mssparkutils.credentials.getToken(audience=\"vault\"),\n",
+        "            expires_on=int(time.time())+60*10 # some random time in future... synapse doesn't document how to get the actual time\n",
+        "        )\n",
+        "\n",
+        "credential = ManagedIdentityCredential()\n",
+        "credential._credential = spoof_token() # monkey-patch the contents of the private `_credential`\n",
+        "\n",
+        "# Set your Key Vault information\n",
+        "KEY_VAULT_URL = \"https://devvault9d194c64.vault.azure.net\"\n",
```

Comment:
Should this be an environment variable of some sort? I'm not sure LAC will have exactly the same vault id.

Reply:
After chatting with Dan this morning, we decided that we need to address this but it's out of scope for this ticket/PR.